gradle/libs.versions.toml

```diff
@@ -30,7 +30,7 @@ firebase-bom = "com.google.firebase:firebase-bom:32.5.0"
 firebase-analytics = { group = "com.google.firebase", name = "firebase-analytics-ktx" }
 firebase-crashlytics = { group = "com.google.firebase", name = "firebase-crashlytics-ktx" }
 firebase-storage = { group = "com.google.firebase", name = "firebase-storage-ktx" }
-firebase-database = { group = "com.google.firebase", name = "firebase-database-ktx" }
+firebase-database = { group = "com.google.firebase", name = "firebase-firestore" }
```

Comment:
I would change the name to `firebase-firestore`, otherwise can be misleading with realtime database. It can also be shorter `firebase-firestore = { module = "com.google.firebase:firebase-firestore" }`

Reply:
I even didn't know about such simplification, thanks 🍻 
Additionally, I've simplified few above firebase dependencies

Reply:
I learned that recently too 😄